<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backup Scheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/sources/Identity/logo-simplified.ico</normaloff>:/sources/Identity/logo-simplified.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleText" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="insights_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Backup Scheduler&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="insights_desc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent;
padding-left: 16px;
padding-right: 16px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose when the backup operation is initiated: on startup, user logon, a scheduled time, or set to &amp;quot;Never&amp;quot; to disable it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainContent">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#mainContent {
	background-color: rgb(255, 255, 255);
    border-top: 1px solid rgb(170, 170, 170);
    border-bottom: 1px solid rgb(170, 170, 170);
    border-left: none;
    border-right: none;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255,255,255)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="initiationGroup" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Initiate:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="initiationType">
               <property name="styleSheet">
                <string notr="true">QComboBox QAbstractItemView {
    background: white; 
    selection-background-color: lightblue;
    selection-color: black;
}


QComboBox QAbstractItemView::item:hover {
    background: lightblue; 
    color: black; 
}
</string>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>Never</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>At startup</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>On a schedule</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>At current user's logon</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="timeSettingsGroup">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background:rgb(255,255,255)
</string>
            </property>
            <property name="title">
             <string>Time Speficic Settings</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item alignment="Qt::AlignTop">
              <widget class="QWidget" name="layout" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_7" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Starting on:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateTimeEdit" name="dateTimeEdit">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="date">
                      <date>
                       <year>2025</year>
                       <month>1</month>
                       <day>1</day>
                      </date>
                     </property>
                     <property name="time">
                      <time>
                       <hour>13</hour>
                       <minute>0</minute>
                       <second>0</second>
                      </time>
                     </property>
                     <property name="currentSection">
                      <enum>QDateTimeEdit::DaySection</enum>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="timeNowCheck">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Now</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item alignment="Qt::AlignTop">
                 <widget class="QWidget" name="recurrenceGroup" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>4</number>
                   </property>
                   <item>
                    <widget class="QComboBox" name="recurrenceType">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QComboBox QAbstractItemView {
    background: white; 
    selection-background-color: lightblue;
    selection-color: black;
}


QComboBox QAbstractItemView::item:hover {
    background: lightblue; 
    color: black; 
}
</string>
                     </property>
                     <property name="duplicatesEnabled">
                      <bool>false</bool>
                     </property>
                     <item>
                      <property name="text">
                       <string>Every</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Once</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="recurrenceStep">
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::UpDownArrows</enum>
                     </property>
                     <property name="specialValueText">
                      <string/>
                     </property>
                     <property name="accelerated">
                      <bool>false</bool>
                     </property>
                     <property name="showGroupSeparator" stdset="0">
                      <bool>false</bool>
                     </property>
                     <property name="suffix">
                      <string/>
                     </property>
                     <property name="minimum">
                      <number>1</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="recurrenceStepUnit">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QComboBox QAbstractItemView {
    background: white; 
    selection-background-color: lightblue;
    selection-color: black;
}


QComboBox QAbstractItemView::item:hover {
    background: lightblue; 
    color: black; 
}
</string>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                     <item>
                      <property name="text">
                       <string>Day/s</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Week/s</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="weekdaysGrid">
                  <property name="title">
                   <string>Initiation Days</string>
                  </property>
                  <layout class="QGridLayout" name="dayChoiceGrid">
                   <property name="topMargin">
                    <number>4</number>
                   </property>
                   <item row="0" column="1">
                    <widget class="QCheckBox" name="mondayChoice">
                     <property name="text">
                      <string>Monday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QCheckBox" name="tuesdayChoice">
                     <property name="text">
                      <string>Tuesday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QCheckBox" name="wednesdayChoice">
                     <property name="text">
                      <string>Wednesday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QCheckBox" name="fridayChoice">
                     <property name="text">
                      <string>Friday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QCheckBox" name="thursdayChoice">
                     <property name="text">
                      <string>Thursday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QCheckBox" name="sundayChoice">
                     <property name="text">
                      <string>Sunday</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QCheckBox" name="saturdayChoice">
                     <property name="text">
                      <string>Saturday</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="dialogButtonsGroup" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QDialogButtonBox" name="dialogButtons">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>timeNowCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
